plugins/plugin-chart-echarts/src/BoxPlot/EchartsBoxPlot.tsx

```diff
@@ -60,8 +60,6 @@ export default function EchartsBoxPlot({
         },
         filterState: {
```

Comment:
Why do we still need `value` in `filterState`? I can't see it being used

Reply:
It's kind of a reserved property in `filterState`, and is used to indicate what the current selection is. If it's unset, e.g. the filter indicator will think that the filter is unset (this will all be documented)

Reply:
Here's with the current master, where neither emitted or received filter indicators are shown:
![image](https://user-images.githubusercontent.com/33317356/117281512-66d70280-ae6c-11eb-9158-d2fe7b98f162.png)
With this change it now looks like this (the indicator is looking at `filterState.value`):
![image](https://user-images.githubusercontent.com/33317356/117282875-e1545200-ae6d-11eb-969f-af9b0374efe9.png)

Reply:
Got it, thanks for explanation!

Reply:
As a future development, I think it might be a good idea to always populate `filterState.value` with a preformatted string. For example, if we have a range filter that's set to the interval  `10000 < x <= 100000000`, and we're using a number formatter that displays `10000` and `100000000` as `10k` and `100M` respectively, a good indicator string would probably be `(10k, 100M]` or `10k < my_col <= 100M`. Now it would just show it as `[10000, 100000000]`, because `filterState.value` is set to an array with that value.